mavros/src/plugins/waypoint.cpp

```diff
@@ -231,11 +386,16 @@ class WaypointPlugin : public plugin::PluginBase {
 
 	bool reschedule_pull;
 
+	bool use_mission_item_int;
+	bool mission_item_int_support_confirmed;
+
 	static constexpr int BOOTUP_TIME_MS = 15000;	//! system startup delay before start pull
 	static constexpr int LIST_TIMEOUT_MS = 30000;	//! Timeout for pull/push operations
 	static constexpr int WP_TIMEOUT_MS = 1000;
 	static constexpr int RESCHEDULE_MS = 5000;
 	static constexpr int RETRIES_COUNT = 3;
+	static constexpr unsigned int MAV_PROTOCOL_CAPABILITY_MISSION_INT = 4;
+
```

Comment:
Use `MAV_PROTOCOL_CAPABILITY::MISSION_INT`.

Reply:
Ah thank you I was looking for that and forgot to look in the mavlink generated files.

Reply:
Please remove that line, as it is not used anymore.